ignite/pkg/relayer/relayer.go

```diff
@@ -58,7 +63,12 @@ func (r Relayer) LinkPaths(ctx context.Context, pathIDs ...string) error {
 }
 
 // Link links chain path to each other.
-func (r Relayer) Link(ctx context.Context, conf relayerconf.Config, pathID string) (relayerconf.Config, error) {
+func (r Relayer) Link(
+	ctx context.Context,
+	session cliui.Session,
```

Comment:
Let's revert this change, cliui only meant to be used in the cmd package.

I actually couldn't understand why do we even pass the session to tsrelayer.Call()

Reply:
The CLIUI is missing in many parts of the code. We still print a lot of things in the default stdout. I'm checking if this is the reason for the stuck UI. I'm doing some tests to figure it out. This is the only thing missing; after fixing it, I will improve the code and remove the PR from draft